benchmarks/src/main/scala/ai/h2o/sparkling/benchmarks/Runner.scala

```diff
@@ -121,10 +121,10 @@ object Runner {
   }
 
   private def filterCollection[T](
-                                   entity: String,
-                                   filter: Option[String],
-                                   collection: Seq[T],
-                                   nameGetter: T => String): Seq[T] = filter match {
+      entity: String,
```

Comment:
It will take me some time to use to it, but I'm fine with it as long it's typical for Scala and used with Spark. I like consistency :)